auth0-lock.ts

```diff
@@ -9,13 +10,20 @@ declare var Auth0Lock: any;
 export class AuthService {
   // Configure Auth0
   lock = new Auth0Lock(this.clientId, this.domain, {
-    closable: false // Users are required to log in
+    closable: false, // Users are required to log in
+    auth: {
+      redirectUrl: (this.redirectPath == null ? location.origin : location.origin + this.redirectPath),

```

Comment:
Turns out this doesn't work like that. For example if `location.origin` is http:\\example.com and `this.redirectPath` isn't set, the results of  `location.origin + this.redirectPath` will end up as __http:\\example.comundefinded__